db/structure.sql

```diff
@@ -9,6 +9,13 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
+--
+-- Name: public; Type: SCHEMA; Schema: -; Owner: -
+--
+
+-- *not* creating schema, since initdb creates it
+
+
```

Comment:
This block is noise created by the version of `pg_dump` you're using and has nothing to do with the actual changes in this PR so should be removed.